main.js

```diff
@@ -1,56 +1,23 @@
-async function hotelsData(url) {
-  const sessionHotels = sessionStorage.getItem('hotels');
-  let hotels;
+const formEl = document.getElementById('formJS');
 
-  if (!sessionHotels) {
-    hotels = await fetch(url)
-      .then((response) => response.json())
-      .then(hotels => hotels);
-    sessionStorage.setItem('hotels', JSON.stringify(hotels));
-  } else {
-    hotels = JSON.parse(sessionHotels);
-  }
-  return hotels;
-}
+formEl.addEventListener('submit', async (event) => {
+  event.preventDefault();
 
-const dataLoves = async () => {
-  const dataMain = await hotelsData('https://fe-student-api.herokuapp.com/api/hotels/popular');
+  const fetchInfo = {
+    method: 'POST',
+    body: new FormData(formEl),
+  };
 
-  const wrapper = document.querySelector('body');
+  const resp = await fetch('https://fe-student-api.herokuapp.com/api/file.', fetchInfo)
```

Comment:
Wrong url. It should be `'https://fe-student-api.herokuapp.com/api/file'`

Reply:
fixed